spec/rails_helper.rb

```diff
@@ -0,0 +1 @@
+require "spec_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.